apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/InMemoryCacheStore.java

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public final class InMemoryCacheStore extends CacheStore {
+
+  private final RecordSet recordSet;
```

Comment:
I feel that `RecordSet` better describes it's purpose (a in memory data structure to store records). 

I don't think we will keep `InMemoryCacheStore` around for very long. I think we will want to ship with some sort `CacheStore` implementation, but I think it will either be disk based or at least a LRU in-memory cache.